File: scala/src/io/stpettersen/blackjack/IUI.scala
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 *
 * Scala Swing/AWT implementation.
 */

package io.stpettersen.blackjack
import java.awt.Graphics

trait IUI {
  def clear: Unit
  def draw(g: Graphics): Unit
}

File: java/io/stpettersen/blackjack/IUI.java
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 *
 * Java Swing/AWT implementation.
*/

package io.stpettersen.blackjack;

import java.awt.Graphics;

public interface IUI
{
    void clear();
    void draw(Graphics g);
}
